pkg/humio/filteralert_transform.go

```diff
@@ -27,6 +27,10 @@ func FilterAlertTransform(hfa *humiov1alpha1.HumioFilterAlert) (*humioapi.Filter
 		filterAlert.ID = hfa.ObjectMeta.Annotations[FilterAlertIdentifierAnnotation]
 	}
 
+	if filterAlert.Labels == nil {
+		filterAlert.Labels = []string{}
+	}
+
 	return filterAlert, nil
```

Comment:
Same as the comment from on the changes to sanitizeFilterAlert. This should not be part of this PR. Same goes for updates to pkg/humio/scheduledsearch_transform.go

Reply:
I do not have any idea why it is showing up on this PR. Perhaps I accidently edited the file. I remove it from the PR.